### PR TITLE
Fix some issues caused by #275

### PR DESCRIPTION
**Pull request by jmcs**
- Fix Main DNS in senza status;
- Delete dangling records when converting to Alias records;

### CODE REVIEW

File: senza/cli.py [hunk 1]
```diff
@@ -952,9 +952,13 @@ def status(stack_ref, region, output, w, watch):
...
                             http_status = 'ERROR'
                     else:
-                        try:
-                            answers = dns.resolver.query(name, 'CNAME')
-                        except:
+                        for record_type in ["CNAME", "A", "AAAA"]:
+                            try:
+                                answers = dns.resolver.query(name, record_type)
+                                break
+                            except:
```
Comment (rafaelcaricio) [inline senza/cli.py]: Really any error?

File: senza/cli.py [hunk 2]
```diff
@@ -952,9 +952,13 @@ def status(stack_ref, region, output, w, watch):
                         except:
                             http_status = 'ERROR'
                     else:
-                        try:
-                            answers = dns.resolver.query(name, 'CNAME')
-                        except:
+                        for record_type in ["CNAME", "A", "AAAA"]:
```
Comment (hjacobs) [inline senza/cli.py]: but this will still not work as the `answer.target.to_text()` will not contain the stack version (only an IP..)

Comment (jmcs) [inline senza/cli.py]: @hjacobs I reverted that change and I'll create a Pull Request later to fix this (maybe by checking the route53 api).

File: senza/manaus/route53.py [hunk 3]
```diff
@@ -337,9 +337,12 @@ def convert_domain_records_to_alias(domain_name: str):
...
     for record in records:
         if record.type != 'A':
-            alias_record = record.to_alias()
             converted_records[record.hosted_zone]['delete'].append(record)
-            converted_records[record.hosted_zone]['upsert'].append(alias_record)
+            try:
+                alias_record = record.to_alias()
+                converted_records[record.hosted_zone]['upsert'].append(alias_record)
+            except ELBNotFound:
+                print("Dangling DNS record  will be deleted: {}".format(record.name))
```
Comment (hjacobs) [inline senza/manaus/route53.py]: extra space between "record" and "will"

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**